README.md

```diff
@@ -78,50 +79,49 @@ Update-MaesterTests
 
 ## Use as GitHub action
 
-Maester is also published to the [GitHub marketplace](https://github.com/marketplace/actions/maester-action) and can be used directly in any GitHub workflow.
+Maester is also published to the [GitHub marketplace](https://github.com/marketplace/actions/run-maester) and can be used directly in any GitHub workflow. Because it is specially built for GitHub it nicely integrates with all the features of GitHub Actions, like uploading artifacts and writing a summary to the workflow run.
 
-Just provide the required client and tenant id. For more details please refer to the [docs](https://maester.dev/docs/monitoring/github/).
+For more details please refer to the [docs](https://maester.dev/docs/monitoring/github/) or the [action repository](https://github.com/maester365/maester-action).
 
 ```yaml
-name: Maester Daily Tests
+name: Daily tests
 
 on:
   push:
     branches: ["main"]
   # Run once a day at midnight
   schedule:
-    - cron: "0 0 * * *"
+    - cron: "0 6 * * *"
   # Allows to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 permissions:
-      id-token: write
+      id-token: write # important because the actions needs it to authenticate to Entra using workload identity
       contents: read
       checks: write
 
 jobs:
-  run-maester-tests:
-    name: Run Maester Tests
+  daily-tests:
+    name: Daily security scan
     runs-on: ubuntu-latest
     steps:
-    - name: Run Maester action
+    - name: Maester 🔥
       id: maester # this is important, by setting the id you can use the output of the action in the next steps
-      uses: maester365/maester@main
+      uses: maester365/maester-action@v1.0.0 # this is the new action
       with:
         client_id: ${{ secrets.AZURE_CLIENT_ID }}
         tenant_id: ${{ secrets.AZURE_TENANT_ID }}
         include_public_tests: true # Optional
+        include_private_tests: true # Optional
         include_exchange: false # Optional
         include_teams: false # Optional
         pester_verbosity: None # Optional - 'None', 'Normal', 'Detailed', 'Diagnostic'
-        mail_recipients: '' # optional a list of email addresses to send the report to
-        mail_userid: '' # optional the user id to use for sending the email
-        include_tags: '' # optional a list of tags to include in the test run
-        exclude_tags: '' # optional a list of tags to exclude in the test run
```

Comment:
When I am working with an unfamiliar action configuration, I find it helpful when optional parameters are included but commented out. For example: 

```yaml
        # mail_recipients: '' # optional a list of email addresses to send the report to
        # mail_userid: '' # optional the user id to use for sending the email
        # include_tags: '' # optional a list of tags to include in the test run
        # exclude_tags: '' # optional a list of tags to exclude in the test run
```

Reply:
And that is why I want to direct everybody to the action documentation. Eventually I think this part has to go from the main readme, opinions?

Reply:
I'm good with that. Directing people to the documentation instead of leaving potential configuration is probably better because those commented optional configs could change in the future. Let's stick with your deletion of those lines.